### PR TITLE
bpo-45212: Add a comment for time.sleep() in tests

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
https://bugs.python.org/issue45212

### CODE REVIEW

**Review by ambv — APPROVED** [review]

File: Lib/test/test_socket.py [hunk 1]
```diff
@@ -6207,6 +6207,7 @@ def _testWithTimeoutTriggeredSend(self):
     def testWithTimeoutTriggeredSend(self):
         conn = self.accept_conn()
         conn.recv(88192)
+        # the wait here needs to be longer than the client-side timeout (0.01s)
```
Comment (vstinner) [inline Lib/test/test_socket.py]:
```suggestion
        # bpo-45212: the wait here needs to be longer than the client-side timeout (0.01s)
```

**Review by vstinner — APPROVED** [review]
LGTM, thank you. I just suggest to mention the bpo to ease future code reviews.

**Comment by vstinner** [comment]
Thank you for adding the comment :-)

**Comment by serhiy-storchaka** [comment]
Thanks Łukasz.

**Pull request merged**